src/pfs/pfs.proto

```diff
@@ -462,6 +462,51 @@ message RunLoadTestResponse {
   google.protobuf.Duration duration = 5;
 }
 
+message ObjectStorageEgressOptions {
+  string url = 1;
+}
+
+message SQLEgressOptions {
+  message FileFormat {
+    enum Type {
+        UNKNOWN = 0;
+        CSV = 1;
+        JSON = 2;
+        PARQUET = 3;
+    }
+    message Options {
+      repeated string json_field_names = 1;
+    }
```

Comment:
reiterating some comments I already made, but I don't think this needs to be nested another level in.